apps/challenges/views.py

```
@@ -1926,20 +1916,22 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
+                    if phase_description_file_path.endswith(
+                        ".html"
+                    ) and isfile(phase_description_file_path):
+                        data["description"] = get_file_content(
+                            phase_description_file_path, "rb"
+                        ).decode("utf-8")
+                    else:
+                        data["description"] = None
+                except PathTraversalError:
+                    raise RuntimeError(
+                        "Challenge configuration contains unsafe file paths."
+                    )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)